requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-pytest==2.8.5
-pytest-cov==2.2.0
-pytest-pep8==1.0.6
--e git+https://github.com/python/mypy@005fd6b357b995f13e3d90293cabf3c8c8c2a952#egg=mypy
+pytest >=4.6.3, <5
+pytest-cov >=2.7.0, <3
+pytest-pep8 >=1.0.6, <2
+mypy >=0.710, <1
```

Comment:
I would fix mypy, since there are often new warnings in the releases, which would then suddenly break CI without a change.

The others are OK that way :)